docs/storage/clone_api.md

```diff
@@ -155,28 +157,18 @@ After the clone is finished, the target can be inspected:
 kubectl get vm vm-clone-target -o yaml
 ```
 
-## Future roadmap
-
-The clone API is in an early alpha version and may change dramatically. There are many improvements and features
-that are expected to be added, the most significant goals are:
-
-* Add more supported source types like `VirtualMachineInstace` in the future.
-* Add a cross-namespace clone support. This needs to be supported for snapshots / restores first.
-
 ### Using clones as a "golden VM image"
 
 One of the great things that could be accomplished with the clone API when the source is of kind `VirtualMachineSnapshot`
 is to create "golden VM images" (a.k.a. Templates / Bookmark VMs / etc). In other words, the following
-workflow would be available:
+workflow is available:
 
 **Create a golden image**
 
 * Create a VM
 
-* Prepare a "golden VM" environment
-
-  * This can mean different things in different contexts. For example, write files, install applications, apply configurations,
-    or anything else.
+* Prepare a "golden VM" environment. This can mean different things in different contexts.
+For example, write files, install applications, apply configurations, or anything else.
 
 * Snapshot the VM
 
```

Comment:
why do you have to delete the vm?

Reply:
I don't :)
Added that it's optional to delete it